lib/Controller/PageController.php

```diff
@@ -94,6 +95,11 @@ public function index(int $announcement = 0): TemplateResponse {
 			$announcement
 		);
 
+		Util::addScript('comments', 'comments');
+		Util::addScript('comments', 'comments-app');
+		Util::addScript('comments', 'comments-tab');
```

Comment:
Not sure whether I get the full picture - but if this is about loading the comments scripts for the sidebar tabs, shouldn't this rather be handled by the [`LoadSidebarScripts` event](https://github.com/nextcloud/server/blob/2e030582b3a156688c7f5d43de39275de2e41894/apps/comments/lib/Listener/LoadSidebarScripts.php) in the comments app?

Reply:
It could but I can not emit the event, because that would load all viewer apps, etc that is visible on files.

Reply:
After some pointing from Robin I noticed the problem or reason for this.
It's that we removed this line: https://github.com/nextcloud/announcementcenter/pull/689/files#diff-4eb824426a8cffb8b33b62b57744c69f32ea30ea755249a7a6170d9402885debL63

bringing it back fixes the problem again :O